python_modules/dagster/dagster/_core/storage/event_log/schema.py

```diff
@@ -137,6 +137,25 @@
     db.Column("create_timestamp", db.DateTime, server_default=get_current_timestamp()),
 )
 
+AssetChecksTable = db.Table(
+    "asset_checks",
+    SqlEventLogStorageMetadata,
+    db.Column(
+        "id",
+        db.BigInteger().with_variant(sqlite.INTEGER(), "sqlite"),
+        primary_key=True,
+        autoincrement=True,
+    ),
+    db.Column("name", db.Text),
+    db.Column("asset_key", db.Text),
+    db.Column("status", db.Text),
+    db.Column("run_id", db.Text),
+    db.Column("result_body", db.Text),
```

Comment:
Should we store the AssetCheckResult nt directly? Should we store the ASSET_CHECK_RESULT event record, which contains the result as event specific data?

I'm partial to the latter since it's what we do now for materializations. We don't need the event storage_id now, but I could imagine needing it in the future